WooCommerce/WooCommerceTests/ViewRelated/Dashboard/Blaze/BlazeCampaignDashboardViewModelTests.swift

```diff
@@ -452,7 +452,8 @@ final class BlazeCampaignDashboardViewModelTests: XCTestCase {
         XCTAssertTrue(sut.shouldRedactView)
     }
 
-    func test_shouldRedactView_is_true_after_reload_finishes_when_only_non_purchasable_product_available() async {
+    // Creating Blaze campaign for non-purchasable product is allowed.
+    func test_shouldRedactView_is_false_after_reload_finishes_when_only_non_purchasable_product_available() async {
```

Comment:
IMO, we can remove this test as we have stopped checking for purchasable condition. As there is no code about purchasable condition it might cause unnecessary confusion. What do you think?

Reply:
Good point, I agree. removed in https://github.com/woocommerce/woocommerce-ios/pull/11671/commits/3633f5afc56a33aa6e3fba55553d144f7e5d5ea5